src/Microsoft.Azure.WebJobs.Extensions.Kafka/Output/KafkaProducerFactory.cs

```diff
@@ -121,7 +127,9 @@ public ProducerConfig GetProducerConfig(KafkaProducerEntity entity)
                 SslKeyLocation = resolvedSslKeyLocation,
                 SslKeyPassword = entity.Attribute.SslKeyPassword,
                 SslCertificateLocation = resolvedSslCertificationLocation,
-                SslCaLocation = resolvedSslCaLocation
+                SslCaLocation = resolvedSslCaLocation,
+                Debug = this.config.Get<KafkaOptions>()?.LibkafkaDebug,
```

Comment:
nit: could get `this.config.Get<KafkaOptions>()` once and reuse in lines 131 and 132

Reply:
Done